CRM/Mailing/Controller/Send.php

```diff
@@ -49,6 +49,10 @@ public function __construct($title = NULL, $action = CRM_Core_Action::NONE, $mod
     }
     if ($mid && !$continue) {
       $clone = civicrm_api3('Mailing', 'clone', ['id' => $mid]);
+      civicrm_api3('Mailing', 'create', [
+        'id' => $clone['id'],
+        'name' => "Copy of " . $clone['values'][$clone['id']]['name'],
```

Comment:
Yup, I think it's better with `Copy of...` instead of `... - Copy`. Matches some similar bits elsewhere.

Actually, borrowing from `CRM/SMS/Form/Group.php` (which also has a `Copy of...`), we should use `ts()`. (Prose should usually be translated.)

```suggestion
        'name' => ts('Copy of %1', [1 => $clone['values'][$clone['id']]['name']]) ,
```